static/js/people.js

```diff
@@ -306,20 +306,22 @@ export function get_full_names_for_poll_option(user_ids) {
     return get_display_full_names(user_ids).join(", ");
 }
 
-export function get_display_full_names(user_ids) {
-    return user_ids.map((user_id) => {
-        const person = get_by_user_id(user_id);
-        if (!person) {
-            blueslip.error("Unknown user id " + user_id);
-            return "?";
-        }
+function map_user_id_to_name(user_id) {
```

Comment:
This function doesn’t map anything. It’s just `get_display_full_name`.

Reply:
renamed to `get_display_full_name`, thanks!